app.js

```diff
@@ -1,15 +1,41 @@
 const express = require("express");
 
-const { getTopics, getAPI } = require("./controller");
+const {
+  getTopics,
+  getArticleById,
+  patchVotesByArticleId,
+} = require("./controller");
 
 const app = express();
 
 app.use(express.json());
 
 app.get("/api/topics", getTopics);
+app.get("/api/articles/:article_id", getArticleById);
+app.patch("/api/articles/:article_id", patchVotesByArticleId);
 
 app.use("*", (req, res) => {
   res.status(404).send({ message: "path not found" });
 });
 
+app.use((err, req, res, next) => {
+  if (err) {
+    res.status(err.status).send({ message: "article not found" });
```

Comment:
As this is error handling middleware there will always be an error of some kind. You should check for something particular on the error, such as `error.status` in this case otherwise you if condition will always be true.